src/web_sys.rs

```diff
@@ -1302,6 +1302,30 @@ impl Context {
             RawRenderingContext::WebGl2(ref gl) => gl.bind_framebuffer(target, Some(framebuffer)),
         }
     }
+
+    // Returns whether the "ldr" profile is support by the `WEBGL_compressed_texture_astc`
+    // extension, with `None` indicating that the extension itself isn't supported.
+    pub fn compressed_texture_astc_get_ldr_profile_support(&self) -> Option<bool> {
+        self.extensions
+            .webgl_compressed_texture_astc
+            .as_ref()
+            .map(|ext| {
```

Comment:
Should we simplify this to return `false` when the extension isn't supported? I don't know if we need to distinguish between the extension not being available here since that could be a separate check anyway.